maven/bnd-export-plugin/src/main/java/aQute/bnd/build/StandaloneRun.java

```diff
@@ -0,0 +1,25 @@
+package aQute.bnd.build;
```

Comment:
We seem to use aQute.bnd.maven as the package root for all code in the maven plugins. Can you change to use the same package as the other classes in this plugin?